package.json

```diff
@@ -5,15 +5,10 @@
     "email": "msopentech@microsoft.com",
     "url": "https://github.com/AzureAD/azure-activedirectory-library-for-js"
   },
-  "licenses": [
-    {
-      "type": "Apache 2.0",
-      "url": "http://www.apache.org/licenses/LICENSE-2.0"
-    }
-  ],
+  "license": "Apache-2.0",
   "repository": {
     "type": "git",
-    "url": ""
+    "url": "https://github.com/AzureAD/azure-activedirectory-library-for-js.git"
   },
   "version": "1.0.8",
```

Comment:
Ah, yes, now I see the new iteration. Thanks.

---
In reply to: [58117639](https://github.com/AzureAD/azure-activedirectory-library-for-js/pull/253#discussion_r58117639) [](ancestors = 58117639)